ProjectManager/ProjectManager/Presentation/TodoListScenes/TodoDetail/ViewModel/TodoDetailViewModel.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  DefaultTodoDetailViewModel.swift
+//  ProjectManager
+//
+//  Created by 김도연 on 2022/07/06.
+//
+
+import Foundation
+import Combine
+
+protocol TodoDetailViewModelInput {
+    func closeButtonDidTap()
+    func doneButtonDidTap(title: String, content: String, deadLine: Date)
+    func editButtonDidTap()
+}
+
+protocol TodoDetailViewModelOutput {
+    var item: Just<TodoListModel> { get }
+    var isCreate: CurrentValueSubject<Bool, Never> { get }
+}
+
+protocol TodoDetailViewModelable: TodoDetailViewModelInput, TodoDetailViewModelOutput {}
+
+struct TodoDetailActions {
+    let dismiss: () -> Void
+}
+
+final class TodoDetailViewModel: TodoDetailViewModelable {
+    
+    // MARK: - Output
+    
+    var item: Just<TodoListModel> {
+        return Just(todoListModel)
+    }
+    
+    let isCreate = CurrentValueSubject<Bool, Never>(true)
+
+    private let todoListModel: TodoListModel
+    private let actions: TodoDetailActions?
+    private let useCase: TodoListUseCaseable
+    
+    init(actions: TodoDetailActions? = nil, useCase: TodoListUseCaseable, todoListModel: TodoListModel) {
+        self.actions = actions
+        self.useCase = useCase
+        self.todoListModel = todoListModel
+        
+        if todoListModel.title.isEmpty && todoListModel.content.isEmpty {
+            isCreate.send(true)
+        } else {
+            isCreate.send(false)
+        }
```

Comment:
```suggestion
        isCreate.send(todoListModel.title.isEmpty && todoListModel.content.isEmpty)
```

반환형이나 넘기는 값이 Bool이면 조건문 자체를 넣어도 됩니다.

Reply:
해당 부분 수정하였습니다!

https://github.com/yagom-academy/ios-project-manager/pull/138/commits/e6f36dc66fb48078842383a5a28f2e7a4ef25cf0